GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -442,6 +451,38 @@ public int compare(String o1, String o2) {
         }
     }
 
+    private void setDocumentProperties(File fileName, final String surveyId)
+            throws FileNotFoundException, IOException {
+        //Set properties on the Workbook. Need the non-streaming interface for this.
+        //TODO: will this increase memory usage? We could GC the big data structures first.
+        FileInputStream fis = new FileInputStream(fileName);
+        XSSFWorkbook workbook = new XSSFWorkbook(fis);
+        POIXMLProperties props = workbook.getProperties();
+
+        /* set some core properties */
+        POIXMLProperties.CoreProperties coreProp = props.getCoreProperties();
+        coreProp.setCreator("Akvo Flow v1.9.25"); //set document creator - not seen in LibreOffice
+        coreProp.setDescription("Form " + surveyId);
+        coreProp.setCategory("Data Cleaning Report"); //category (seen in custom properties in Calc)
+
+        /* some custom properties */
+        POIXMLProperties.CustomProperties custProp = props.getCustomProperties();
+        custProp.addProperty("Version", "Akvo Flow v1.9.25");
+        custProp.addProperty("Generated", new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(new Date()));
+        
+        FileOutputStream fos = new FileOutputStream(fileName);
+        workbook.write(fos);
+        fos.close();
+    }
+
+    private void addColumnWarnings(Workbook wb) {
```

Comment:
As this is unused for now, I think we should also remove till we get round to using it.